contents/handbook/engineering/incidents.md

```diff
@@ -98,9 +98,12 @@ Don't forget to disable the in-app banner too.
 
 24-48 hours after an incident, we should have a quick sync meeting with a small group of people involved in the incident, plus Tim. If you raised the incident, you can schedule this. If you don’t want to or can’t, Tim is happy to take over at this point, just let him know.
 
-Incident.io will automatically create an incident analysis doc that you can paste into a PR against the [incidents analysis repository](https://github.com/PostHog/incidents-analysis). It'll have a timeline of everything that happened.
+Incident.io will automatically create an incident analysis doc that you can paste into a PR. 
```

Comment:
I don't think this is true. It was fine before as the key point was true
```suggestion
Before this meeting you should create a PR against the [incidents analysis repository](https://github.com/PostHog/incidents-analysis). You can use Incident.io's automatic analysis as a starting point. In general it is advised to post short point in time summaries of actions taken in the incident Slack channel so that you can more easily fill this out after the fact.
```